vulture_allowlist.py

```diff
@@ -134,6 +137,8 @@
 
 ILinuxAgentCommandBuilder.build_download_command
 ILinuxAgentCommandBuilder.build_run_command
+ILinuxAgentCommandBuilder.build_permission_change_command
+ILinuxAgentCommandBuilder.permission_change_options
```

Comment:
We should remove vulture from this repository

Reply:
Probably. The right way to go about it is to make it an option in the [cookiecutter template](https://github.com/guardicode/infection-monkey-cookiecutter/) and then use cruft to update this repo and not include it.